subprojects/docs/src/docs/userguide/authoring-builds/multi_project_builds.adoc

```diff
@@ -391,7 +391,7 @@ include::{snippetsPath}/multiproject/partialTasks/tests/multiprojectPartialTasks
 ----
 ====
 
-The build is executed from the `water` project. Neither `water` nor `tropicalFish` have a task with the name `distanceToIceberg`. Gradle does not care. The simple rule mentioned already above is: Execute all tasks down the hierarchy which have this name. Only complain if there is _no_ such task!
+The build is executed from the `water` project. Neither `water` nor `tropicalfish` have a task with the name `distanceToIceberg`. Gradle does not care. The simple rule mentioned already above is: Execute all tasks down the hierarchy which have this name. Only complain if there is _no_ such task!
```

Comment:
Why no kebab for this one?

Reply:
all lowercase is kebab :) 
With all seriousness, I've done this for consistency: there was already a project called `bluewhale` (and not `blueWhale`).

Reply:
I think all fish and water references should really be remove in favour of samples based on real use cases. So I think this really does not matter as we will hopefully update this part of the docs soon.